### ISSUE TITLE
css-loaders build not works 

### ISSUE THREAD

**Issue by zjhiphop** [vuejs-templates/webpack#33]
1. First install 'node-sass' and 'sass-loader'
2. Then in the project main.js, import scss file, like:
```
reqire('path-to/test.scss')
```

Then when i run 'npm run build', there always expose an error: 
```
ERROR in ./src/scss/app.scss
Module parse failed: /Users/jade/Documents/project/workspace/hk/online_jd/wapdev/paotuibang-new/src/scss/app.scss Line 2: Unexpected token ILLEGAL
You may need an appropriate loader to handle this file type.
| /** IMPORTS **/
| @import "./variables.scss";
|
| /** GLOBALS **/
 @ ./src/main.js 31:0-26
```

For simple solution is to add 'scss' loader config inside webpack config file. But how to integrate it with 'css-loaders'?

**Comment by chrisvfritz** [comment]
@zjhiphop Check out [the docs for `sass-loader`](https://github.com/jtangelder/sass-loader#apply-via-webpack-config). You'll need to a loader for SCSS to use them outside of components. The only difference is that instead of the `style` loader, you'll use `vue-style`. Let me know how that works out for you. :smiley:

**Comment by chrisvfritz** [comment]
@zjhiphop Just checking in. Have you had a chance to see if the suggestion above worked for you?

**Comment by chrispappas** [comment]
I am trying to do this literally right now - saw your last comment pop up in real time!

If you can help with an example snippet that would be helpful. I'm trying to do:

```
require('src/assets/scss/main.scss')
```
 inside my App.vue component

**Comment by chrispappas** [comment]
I actually might be on the right track here already, added the following to my `webpack.base.conf.js` file in the `loaders` array (although getting other unrelated errors so I can't quite confirm):

```
      {
        test: /\.scss$/,
        loaders: ["css", "sass"]
      },
```

**Comment by chrisvfritz** [comment]
@chrispappas _Almost_. I believe this would be what you want:

``` scss
{
  test: /\.scss$/,
  loaders: [ 'vue-style', 'css', 'sass' ]
}
```

**Comment by chrispappas** [comment]
Nice, I had actually added the stock style-loader and prepended `style` to the loaders array, which seems to work great. Should I replace it with `vuejs/vue-style-loader` instead? This seems to work just fine for my purposes at this point with the stock `style-loader` plugin.

**Comment by chrisvfritz** [comment]
@yyx990803 forked `style-loader` to fix some issues with it. I don't know _a lot_ about that - I've just always used `vue-style-loader` and never had issues. :smiley: Evan will be able to better explain the history behind that decision and whether it matters for you in this case.

**Comment by yyx990803** [comment]
I forked it to provide better source map support for injected style tags during development, but other than that they are largely the same.

**Comment by chrispappas** [comment]
Thanks for the help guys, much appreciated.

**Comment by chrisvfritz** [comment]
@chrispappas Excellent. Even though this is resolved, I want to keep the issue open for now. I'm considering implementing [something similar to this](https://github.com/HenrikJoreteg/hjs-webpack/blob/master/lib/installed-style-loaders.js), so that CSS preprocessor files just automatically work if the loader is installed, similar to how they do in components.

**Comment by chrisvfritz** [comment]
Resolved in #84.
